ci/env/kava-internal-testnet/genesis.json

```diff
@@ -3307,10 +3325,28 @@
             ],
             "active": true
           },
+          {
+            "market_id": "dai:usd",
+            "base_asset": "dai",
+            "quote_asset": "usd",
+            "oracles": [
+              "kava1hdn83q3srldcpan4ex2v4npuae8hmnfrps3e6h"
+            ],
+            "active": true
+          },
+          {
+            "market_id": "dai:usd:30",
+            "base_asset": "dai",
+            "quote_asset": "usd",
+            "oracles": [
```

Comment:
Do we have oracle for internal testnet? Or is it just a required value?

Reply:
Good question, I'm not sure where the private key for `kava1hdn83q3srldcpan4ex2v4npuae8hmnfrps3e6h` is defined 🤔

Reply:
We just hard code the prices in the pricefeed and set them to never expire: https://github.com/Kava-Labs/kava/blob/ee4fc56f013c06a093f3b5a1202652d7e1e81124/ci/env/kava-internal-testnet/genesis.json#L3555